interview-questions.md

```diff
@@ -6,38 +6,50 @@ First, without external resources. Challenge yourself to answer from memory.
 
-Researched answer:
+Researched answer: I couldnt find any online sources that informed me of the required routes that needed params
 
 3. Name three rails generator commands. What is created by each?
 
-Your answer:
+Your answer: rails g model, this will create a table for rails that can be fillable with different instances. rails g resources, does similair function to rails g model but removes view by default for backend developing. i cant think of any others at the top of my head
 
-Researched answer:
+Researched answer: rails generate -h would give a list of available generate commands to use. -h is listed as the help option. there is rails generate - f which forces the terminal to write over past saved files. rails g -q will suppress status output 
+source: https://medium.com/@josephdlawson21/using-the-generate-command-in-rails-9c738380f2d9
 
 4. Consider the Rails routes below. What is the name of the controller method that would be called by each route? What action would each of the controller methods perform?
 
-action: "GET" location: /students
+action: "GET" location: /students   index.controller it would show list of students
 
-action: "POST" location: /students
+action: "POST" location: /students update.controller it would allow creation of students info
 
-action: "GET" location: /students/new
+action: "GET" location: /students/new index.controller 
 
-action: "GET" location: /students/2
+action: "GET" location: /students/2 index.controller looks at one of the students info
 
-action: "GET" location: /students/2/edit
+action: "GET" location: /students/2/edit update.controller allow edit or update of student information
 
-action: "PATCH" location: /students/2
+action: "PATCH" location: /students/2 update.controller allow edit or update of student information
 
-action: "DELETE" location: /students/2
+action: "DELETE" location: /students/2 delete.controller would delete information of students
 
 5. As a developer, you are making an application to manage your to do list. Create 10 user stories that will help you get your application started. Read more about [user stories](https://www.atlassian.com/agile/project-management/user-stories).
+
+story 1: I want to be able to write the things i need to so and save them 
+story 2: I want to have my list be able to have a pink background or have the option to
+story 3: i want to be able to put the lists in a table where each column is named for each day of the week. This will allow me to keep track of chores or appointments on certain days
+story 4: The application will have a title at the top saying to do list application in bold so it is easily distinguishable
+story 5: I will be able to underline text i make and/or bolden them for important taks
+story 6:  i will be able to use emoji's in the to do list 
+story 7: It will be compatible with text-to speech software in the event that I cant see
+story 8: will have an enlarge function so I can see without glassess
```

Comment:
Ooo, I can see you have some interest in making this accessible! Excellent work on that!  Just keep in mind that there are differences between a user story and a developer story.  User stories will be focused on the user experience whereas the developer stories will talk about the more specific development details like databases.